update-pillow-tag.sh

```diff
@@ -0,0 +1,14 @@
+if [ $# -eq 0 ]; then
+    echo "Usage: update-pillow-tag.sh [[release tag]]"
+    exit
+fi
+
+git checkout master
+git submodule init
+git submodule update Pillow
+cd Pillow
+git fetch --all
+git checkout $1 || exit
```

Comment:
```suggestion
git checkout $1
```

We could remove the exit and have the script immediately exit on any error by adding this near the top: 
```bash
set -e # exit on error
```
https://stackoverflow.com/q/19622198/724176

And it's good to start with a shebang: 
```bash
#!/bin/bash
```
https://stackoverflow.com/a/10383546/724176

Reply:
Okay, changes made